webrtc.html

```diff
@@ -4166,7 +4166,7 @@ <h2 id="sec.stats-model">Statistics Model</h2>
 
       <dl class="idl" title="partial interface RTCPeerConnection">
         <dt>Promise&lt;RTCStatsReport&gt; getStats(
-          optional MediaStreamTrack? selector)</dt>
+          optional MediaStreamTrack? selector = null)</dt>
```

Comment:
In this case then it is either optional, nullable and a default of null; or it is merely optional. The difference being that the former permits an explicit null value whereas the latter would require omission to convey the same meaning. I prefer the option that permits only one way. Just make it optional.  ...or use overloads so that you can highlight the fact that these are very very different APIs depending on what is passed in.